resources/migrations/000_migrations.yaml

```diff
@@ -13169,6 +13170,9 @@ databaseChangeLog:
               WHERE c.personal_owner_id IS NULL;
         - sql:
             dbms: h2
+            # I would have preferred using MERGE ... USING instead of WHERE EXISTS ... but it's broken in 1.4.197
+            # because of https://github.com/h2database/h2database/issues/1034, which is fixed in 1.4.198. So this will
```

Comment:
can we not bump h2 to `1.4.198` for this then?

Reply:
I don't want to risk it. We tried bumping to `1.4.200` a while back and it turns out that it's not 100% backwards compatible with `1.4.197` files. I don't know if `1.4.198` is fine or not but I don't feel like finding out 2 weeks before RC